lib/graphql/schema/resolver.rb

```diff
@@ -210,6 +222,23 @@ def load_application_object_failed(err)
         raise err
       end
 
+      # Call this during loading or authorization to
+      # skip {#resolve} and return data instead of top-level errors.
+      #
+      # It doesn't halt the flow of the code so be sure to return
+      # after calling it, or combine it with `return return_with(...)`.
+      #
+      #
+      # @param response [Object] The value to respond with
+      # @return [void]
+      def return_with(response)
```

Comment:
was this a previous idea? if it's still on the table, i think `resolve_with` might fit the GraphQL nomenclature nicely.

Reply:
oops, oh yeah, it can be removed now since `return` is used instead